attest/untrusted/src/lib.rs

```diff
@@ -67,8 +67,13 @@ impl QuotingEnclave {
     }
 
     pub fn target_info() -> Result<(TargetInfo, EpidGroupId), TargetInfoError> {
-        let gid = Self::epid_group_id()?;
-        Ok((DcapQuotingEnclave::target_info()?, gid))
+        let mut qe_info = TargetInfo::default();
```

Comment:
The other `target_info` was using the DCAP interface which requires a full DCAP machine. Reverted back to the previous EPID implementation